### DISCUSSION TITLE
Docker compose error

### DISCUSSION THREAD

**Discussion by Remraf1** [cmu-sei/GHOSTS#94] [General]
After the "docker-compose up" command I am getting a "0.0.0.0:3000 bind address already in use" error.  When I do a "docker-compose ps" command I see api and postgres up, and grafana down, exit 128 state.  

Any recommendations are appreciated, first attempt setting up a GHOSTS instance.  Thank you!

**Comment by sei-dupdyke** [comment]
Hmm, is something else running on 3000?

We can work around this issue by:

1. `docker-compose down` to bring down the stack
2. Change [this line](https://github.com/cmu-sei/GHOSTS/blob/master/src/Ghosts.Api/docker-compose.yml#L53) in the compose file from `'3000:3000'` to `'3001:3000'`
3. `docker-compose up -d` to bring the system back up
4. point your web browser now to `http://localhost:3001`

LMK if this gets you around it - of course, you can use any port of your choice that is not in use in lieu of 3001.

**Reply by Remraf1** [reply]
That looks like it fixed it, thank you sir!

Another question:  I don't see the v6 windows/linux clients in the files, will I need to compile them?

**Reply by sei-dupdyke** [reply]
Awesome that it works!

You can get compiled binaries here --> https://github.com/cmu-sei/GHOSTS/wiki/Installation-from-distribution-binaries

**Comment by Remraf1** [comment]
I got it to work thank you!  For some reason the download link comes up as 404 on my work computer, but was able to download it at home.

**Reply by sei-dupdyke** [reply]
Excellent that you got it resolved. Curious why that same link was 404 at work, but hey, I have some of those same issues periodically ;)
